fastlane/Fastfile

```diff
@@ -537,6 +540,7 @@ ENV["APP_STORE_STRINGS_FILE_NAME"]="AppStoreStrings.pot"
       "app_store_screenshot-3"    => {desc: "app_store_screenshot_3.txt"},
       "app_store_screenshot-4"    => {desc: "app_store_screenshot_4.txt"},
       "app_store_screenshot-5"    => {desc: "app_store_screenshot_5.txt"},
+      "app_store_screenshot-6"    => {desc: "app_store_screenshot_6.txt"},
```

Comment:
If we do have a new `app_store_screenshot_6.txt`, shouldn't `fastlane/screenshots.json` also have changed in that PR accordingly? I don't see the related change there in this branch though?

Reply:
Good call – this is no longer necessary :) 

Removed in 5c32829e2945b3654c49e0e5d23471270cf69698